relude.cabal

```diff
@@ -142,6 +142,7 @@ library
                            Relude.Extra.Newtype
                            Relude.Extra.Tuple
                            Relude.Extra.Type
+                           Relude.Extra.Types
```

Comment:
I think that there is no need to introduce new `Relude.Extra.Types` module. Let's just put all these functions into `Relude.Extra.Type`

Reply:
Sorry, I took title in issue literally :smile:...